### PR TITLE
fix(*): fix flow union props rendering

### PR DESCRIPTION
**Pull request by stepancar**
https://github.com/styleguidist/react-styleguidist/issues/1032

### CODE REVIEW

**Comment by sapegin** [comment]
Cool, thanks! Could you please have a look if we can add some tests here? 

> 83.33% of diff hit (target 97.56%)

File: src/rsg-components/Props/util.js [hunk 1]
```diff
@@ -15,7 +15,20 @@ export function unquote(string) {
  * @returns {object}
  */
 export function getType(prop) {
-	return prop.flowType || prop.type;
+	if (prop.flowType) {
+		if (
```
Comment (sapegin) [inline src/rsg-components/Props/util.js]: Could you please add a comment with an example how this looks in Flow?

Comment (stepancar) [reply src/rsg-components/Props/util.js]: @sapegin hm, https://github.com/styleguidist/react-styleguidist/pull/1035/files#diff-6a050c2c62e825106dba14a944eda318R369

Comment (sapegin) [reply src/rsg-components/Props/util.js]: It's hard to connect a test with a with a particular line branch of code ;-)

**Pull request merged**